hooks/syslog/syslog.go

```diff
@@ -4,10 +4,8 @@ package syslog
 
 import (
 	"fmt"
-	"log/syslog"
-	"os"
-
 	"github.com/sirupsen/logrus"
+	"log/syslog"
```

Comment:
please remove this formatting change, github.com/sirupsen/logrus imports should be formatted at the end with a space between them and std lib imports.